book/src/development/infrastructure/changelog_update.md

```diff
@@ -56,6 +56,29 @@ and open that file in your editor of choice.
 When updating the changelog it's also a good idea to make sure that `commit1` is
 already correct in the current changelog.
 
+#### From commit range to PR link
+
+Raw commit ranges aren't used, they aren't very useful or illustrative, so we developed the concept of PR ranges to help the user understand the size of a new update.
```

Comment:
hm, it's less useful to mention the old model here.

Let's just document what we do, rather than what we used to do